services/user.py

```diff
@@ -0,0 +1,58 @@
+from django.contrib.auth import get_user_model
+
+
+def create_user(
+        username: str,
+        password: str,
+        email=None,
+        first_name=None,
+        last_name=None
+):
+    if not email:
+        email = ""
+
+    if not first_name:
+        first_name = ""
+
+    if not last_name:
+        last_name = ""
+
+    return get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        email=email,
+        first_name=first_name,
+        last_name=last_name
+    )
```

Comment:
1) function should only create `user`, and `return` nothing
2) rewrite `if` conditions